OJP/OJP_ModesSupport.xsd

```diff
@@ -82,12 +113,43 @@
 			</xs:element>
 		</xs:sequence>
 	</xs:complexType>
-	<xs:simpleType name="ContinuousModesEnumeration">
+	<xs:simpleType name="ContinuousModesOfOperationEnumeration">
 		<xs:annotation>
-			<xs:documentation>Types of cmodes that  run at any time without a timetable.</xs:documentation>
+			<xs:documentation>Types of MODES OF OPERATION that run at any time without a timetable and that are not classic flexible services.</xs:documentation>
 		</xs:annotation>
 		<xs:restriction base="xs:string">
 			<xs:enumeration value="walk"/>
+			<xs:enumeration value="commuterCarPooling">
+				<xs:annotation>
+					<xs:documentation>MODE OF OPERATION from NeTEx personal.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="demandResponsive">
+				<xs:annotation>
+					<xs:documentation>area based demand responsive service (without SCHEDULED STOP POINTs).</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="ownCar">
+				<xs:annotation>
+					<xs:documentation>MODE OF OPERATION from NeTEx personal.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="privateLift">
+				<xs:annotation>
+					<xs:documentation>MODE OF OPERATION from NeTEx personal.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="TimedModesOfOperationEnumeration">
+		<xs:annotation>
+			<xs:documentation>Types of MODES OF OPERATION  are scheduled, classic and also flexible. From NeTEx.</xs:documentation>
```

Comment:
Do you want to mention where from NeTEx this is here too?

Reply:
no longer referencing NeTEx but the Transmodel discussion.